staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -349,6 +349,13 @@ type ListOptions struct {
 	// add, update, and remove notifications. Specify resourceVersion.
 	// +optional
 	Watch bool `json:"watch,omitempty" protobuf:"varint,3,opt,name=watch"`
+	// AllowBookmarks opts-in for receiving watch events with type Bookmark.
+	// Bookmark events doesn't have any guarantees and it is up-to a server
+	// to decide when such events should be sent.
+	// If Watch field is unset, this field is ignored.
```

Comment:
This needs to be feature gated and declared as alpha here.

Reply:
> If the feature gate `WatchBookmarks` is not enabled this field is ignored.

Reply:
Actually... I'm tempted to say that if the feature gate is off the watch bookmarks call should return an error, but we haven't done that before.

Reply:
> I'm tempted to say that if the feature gate is off the watch bookmarks call should return an error

goal if the feature gate is off is for the server to behave as much as possible like a server without the feature. we should not error on the call.